app/assets/stylesheets/welcome.scss

```diff
@@ -1,25 +1,66 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
-$backdrop : #C9D6AE;
+$backdrop-black : rgba(0, 0, 0, 0.76);
+$label-blue: #008CBA;
 
 .welcome {
   background-image: image-url('skyscrapers.jpg');
+  background-position: bottom;
   background-repeat: no-repeat;
   -webkit-background-size: cover;
   -moz-background-size: cover;
   -o-background-size: cover;
   background-size: cover;
   min-height: 100%;
-  min-width: 1024px;
+  min-width: 66%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   }
 
+  #bottom_right_sign_up, #bottom_right_sign_in {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    padding: 25px;
+    font-size: 18px;
+  }
+
+.login {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100%;
+}
+
+#partial {
+  width: 70%;
+}
+
 .top-bar {
   position-left: 0;
   position-right: 0;
   width: 100%;
 }
 
+#user_email, #user_password, #user_first_name, #user_last_name, #user_phone, #user_password_confirmation {
+  border-bottom: line;
```

Comment:
Properties should be ordered background-color, border-bottom, border-left, border-right, border-top, box-shadow, font-size, height